src/com/google/javascript/jscomp/ProcessCommonJSModules.java

```diff
@@ -878,7 +878,7 @@ private void expandObjectLitAssignment(NodeTraversal t, Node export) {
           } else {
             value = IR.name(key.getString());
           }
-        } else if (key.isMemberFunctionDef()) {
+        } else if (key.isMemberFunctionDef() || key.isGetterDef()) {
```

Comment:
This blocks the crash, but doesn't produce correct output.  Either of the following would be acceptable transformations:

```js
/** @const */ var module$test = {
  get a() { return 1; }
};
```

or

```js
Object.defineProperties(module$test, {
  a: {
    get: function() { return 1; };
  }
});
```